### PR TITLE
add link to youtube channel in navbar

### PR DESCRIPTION
**Pull request by PierreBesson**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -248,19 +248,14 @@ <h2 class="heading">JHipster news</h2>
...
+                            <li><b>The JHipster Developers Association</b> now hosts <b>Virtual Meetups</b> on its <a href="https://www.youtube.com/c/JHipsterDevelopersAssociation">Youtube Channel</a>.</li>
+			                <li><b>Full Stack Development with JHipster - Second edition</b> by Deepu K Sasidharan and Sendil Kumar is published. Get it on <a href="https://www.packtpub.com/web-development/full-stack-development-with-jhipster-second-edition" target="_blank" rel="noopener">Packt</a> and <a href="https://smile.amazon.com/Full-Stack-Development-JHipster-microservices/dp/1838824987" target="_blank" rel="noopener">Amazon</a>. Get 25% discount on E-books from Amazon and Packt using discount code "25JAVASILVER". Valid until 1st of July 2020</li>
```
Comment (pascalgrimaud) [inline index.html:258]: indent?

Comment (PierreBesson) [reply index.html:258]: I personally don't see the indent problem.

Comment (pascalgrimaud) [inline index.html:258]:
```suggestion
                            <li><b>Full Stack Development with JHipster - Second edition</b> by Deepu K Sasidharan and Sendil Kumar is published. Get it on <a href="https://www.packtpub.com/web-development/full-stack-development-with-jhipster-second-edition" target="_blank" rel="noopener">Packt</a> and <a href="https://smile.amazon.com/Full-Stack-Development-JHipster-microservices/dp/1838824987" target="_blank" rel="noopener">Amazon</a>. Get 25% discount on E-books from Amazon and Packt using discount code "25JAVASILVER". Valid until 1st of July 2020</li>
```

**Review by pascalgrimaud — CHANGES_REQUESTED** [review]

**Comment by PierreBesson** [comment]
Thanks @pascalgrimaud for the tab -> space correction.

**Pull request merged**